db/questions.yml

```diff
@@ -271,22 +269,27 @@ other_electrical_problem:
 window_problem:
   question: Is your problem one of these?
   answers:
-    - text: Window is not shutting / locking (above ground floor)
+    - text: Window is not shutting / locking properly - uPVC
 
-    - text: Minor damage to glazing
+    - text: Window is not shutting / locking properly - Wood
+
+    - text: Damage to glazing
 
     - text: Something else
       desc: window_problem
 
 
 external_doors_problem:
-  question: Are you able to secure access to your property?
+  question: Is your problem one of these?
   answers:
-    - text: 'Yes'
-      desc: external_doors_problem
+    - text: Front or back door is stiff or loose - uPVC
```

Comment:
Not sure if this would be better as two questions:

`Front or back door is stiff of loose` -> `What is your door made of?`

The second question could have `Wood` / `uPVC` and `I'm not sure` as answers, the latter directing them to the call centre for assistance(?)

Reply:
I'm being quite frank here with the identifying of material, let me test this with residents first rather than adding an additional question.